script/gen_requirements_all.py

```diff
@@ -140,6 +140,7 @@
     "pyps4-homeassistant",
     "pyqwikswitch",
     "PyRMVtransport",
+    "pysaj",
```

Comment:
This is only needed if you want to write tests.

Reply:
Thanks for the info, will remove it and rerun `script.gen_requirements_all`